src/utils.jl

```diff
@@ -221,7 +221,7 @@ macro matchable(expr)
         $expr
         SymbolicUtils.iscall(::$name) = true
         SymbolicUtils.operation(::$name) = $name
-        SymbolicUtils.arguments(x::$name) = getfield.((x,), ($(QuoteNode.(fields)...),))
+        SymbolicUtils.children(x::$name) = getfield.((x,), ($(QuoteNode.(fields)...),))
```

Comment:
as well as this

Reply:
ahhh, yes, that makes sense. missed the `iscall(::$name) = true` 👍

Reply:
Also missing `head`